index.html

```diff
@@ -84,7 +83,163 @@ <h1 class="primary py-2 my-2">
       </div>
     </section>
 
-    <
+    <section class="grid text-center m-4 p-2 projects">
+      <h1 class="primary py-2 my-4 text-center">My Recent Work</h1>
+
+      <div class="card my-2">
+        <div class="card-body d-flex flex-column justify-content-center">
+          <p class="placeholder-glow">
+            <span
+              class="placeholder col-12 bg-primary project-sample-img"
+            ></span>
+          </p>
+
+          <h5 class="card-title">Datafy</h5>
+          <ul>
+            <li class="badge text-bg-secondary">REACT</li>
+            <li class="badge text-bg-secondary">Big Query</li>
+            <li class="badge text-bg-secondary">Shopify API</li>
+            <li class="badge text-bg-secondary">
+              Google Cloud Platform Services
+            </li>
+            <li class="badge text-bg-secondary">ExpressJS</li>
+            <li class="badge text-bg-secondary">MySQL</li>
+          </ul>
+          <p class="card-text">
+            App that interacts with the Shopify API to import statistics to a
+            series of Google Cloud Platform services, to analyze the data and
+            present it to the user using a REACT frontend.
+          </p>
+          <a href="#" class="btn btn-primary teal-button">See Project</a>
```

Comment:
- [ ] K kindly make sure your button looks exactly as the one on the provided Figma design. 

   - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/200127183-04a8657a-b1c8-4efc-aef0-934cc36fd4bb.png)

   - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/200127200-083ec14d-5cd1-4058-b037-ae66dfb592d4.png)

Reply:
Fixed
![image](https://user-images.githubusercontent.com/44822765/200138434-79220dd2-edd2-4cde-b84d-7130e8bf3661.png)